src/components/portfolio/Dashboard.tsx

```diff
@@ -85,12 +203,17 @@ const Dashboard: React.FC<DashboardProps> = ({ token, selectPortfolio }) => {
             }
           />
         )}
-        {portfolios && (
+        {state.portfolios && (
```

Comment:
Do we need a state where we consider no errors, no portfolio?

Reply:
`state.errors == undefined && state.portfolios == undefined` is only true while loading the portfolios. If the user has no portfolios then `state.portfolios == []`. `[]` is truthy, so in this case the `PortfolioOverview` component is rendered and shows a message that the user has no portfolios instead of the table or cards view.